ui/lib/core/addon/components/policy-example.hbs

```diff
@@ -6,15 +6,16 @@
 <div class="has-bottom-margin-s">
   {{#if (eq @policyType "acl")}}
     <p data-test-example-modal-text="acl">
-      ACL Policies are written in Hashicorp Configuration Language (
-      <ExternalLink @href="https://github.com/hashicorp/hcl">HCL</ExternalLink>
```

Comment:
Great work replacing these! Sometimes we don't want to remove our components completely and instead want to use HDS components _within_ our custom components. In this case, I think replacing is the right move! Since `ExternalLink` doesn't do much more of us and using the HDS component directly is more seamless (in my opinion).

Here's an example of where we used the HDS component inside a custom component https://github.com/hashicorp/vault/blob/main/ui/lib/core/addon/components/page/breadcrumbs.hbs#L6

This is definitely case-by-case and depends on on the usage. With the breadcrumbs it was faster to update the custom component instead of replace **every** instance of breadcrumbs, plus we have some standardization built in to make our usage more consistent. Eventually, we may decide the custom breadcrumb component doesn't serve us anymore, so this sort of thing is always evolving